src/Engine/ProtoCore/SyntaxAnalysis/AstVisitor.cs

```diff
@@ -0,0 +1,400 @@
+﻿using System;
```

Comment:
Not only N2C. For example, we do identifier mapping in code block node, that mapping could use visitor pattern.